src/app/Plans/components/Wizard/StorageMapping.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import {
+  Checkbox,
+  Radio,
+  TextContent,
+  Text,
+  TextInput,
+  TextVariants,
+  FormGroup,
+} from '@patternfly/react-core';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+
+import SimpleSelect, { OptionWithValue } from '@app/common/components/SimpleSelect';
+import { IStorageMapping } from '@app/Mappings/types';
+import AddEditStorageMapping from '@app/Mappings/Storage/components/AddEditStorageMapping';
+
+interface IStorageMappingProps {
+  storageMappingList: IStorageMapping[];
+}
+
+const StorageMapping: React.FunctionComponent<IStorageMappingProps> = ({
+  storageMappingList,
+}: IStorageMappingProps) => {
+  const storageMappingOptions = Object.values(storageMappingList).map((storageMapping) => ({
+    toString: () => storageMapping.name,
+    value: storageMapping,
+  })) as OptionWithValue<IStorageMapping>[];
+
+  const [isUseExistingMappingChecked, toggleUseExistingMapping] = React.useReducer(
+    (isUseExistingMappingChecked) => !isUseExistingMappingChecked,
+    true
+  );
+
+  const [isUseNewMappingChecked, toggleUseNewMapping] = React.useReducer(
+    (isUseNewMappingChecked) => !isUseNewMappingChecked,
+    false
+  );
+
+  const [isSaveNewStorageMapping, toggleSaveNewStorageMapping] = React.useReducer(
+    (isSaveNewStorageMapping) => !isSaveNewStorageMapping,
+    false
+  );
+
+  const [storageMapping, setStorageMapping] = React.useState<IStorageMapping | null>(null);
+  const [addStorageMappingName, setAddStorageMappingName] = React.useState<string>('');
+
+  const onChange = () => {
+    toggleUseNewMapping();
+    toggleUseExistingMapping();
+  };
+
+  return (
+    <>
+      <TextContent className={spacing.mbMd}>
+        <Text component={TextVariants.h4}>
+          Select an existing storage mapping between your source and target providers, or create a
+          new one.
+        </Text>
+      </TextContent>
+      <Radio
+        id="existing-mapping"
+        label="Use an existing storage mapping"
+        name="radio-existing"
+        onChange={onChange}
+        isChecked={isUseExistingMappingChecked}
+      />
+      {isUseExistingMappingChecked && (
+        <FormGroup
```

Comment:
If you wrap this whole thing in a `<Form>` the CSS will add some spacing between your `<FormGroup>s`

Reply:
@mturley, thanks.